composer.json

```diff
@@ -15,22 +15,20 @@
     ],
     "config" : {
         "platform": {
-            "php": "5.3.9"
+            "php": "7.2.5"
```

Comment:
This means phpbrew will drop the support before php 7.2?

Reply:
yes, pimple its support 8.1 version is  required  only 7.2.5 higher
https://github.com/silexphp/Pimple/blob/69bba50d1600342deae9dca3372a2b5f1ac32ae6/CHANGELOG#L3

Reply:
Pimple's update is required for the PHP 8.1 support?

Reply:
yes

Reply:
https://github.com/silexphp/Pimple/commit/561cef41326493c29083e5e7d177a7fafd7cf453

Reply:
@c9s, Pimple versions require PHP version are different after checking the [packagist package URL](https://packagist.org/packages/pimple/pimple#v3.2.3). Some situations are as follows:

- `v3.2.3` requires `php-5.3` version at least.
- After `v3.2.3` version is released, it requires the `php-7.2` version at least.

I saw the [CLIFramework](https://github.com/c9s/CLIFramework/blob/3.0/composer.json#L19) uses the `"pimple/pimple": "*"` and I think it's fine to use the config to be the `5.3.9`.

Once we upgrade to the `corneltek/cliframework` to be `4.1.2`, we can consider using the config setting to be the PHP `5.5` version because the `corneltek/cliframework` `4.1.2` version requires the PHP `5.5.0` version at least.